### ISSUE TITLE
Child content within `<d2l-alert>` is not accessible by default

### ISSUE THREAD

**Issue by Kiefer-Co-D2L** [BrightspaceUI/core#5015]
Regarding [`<d2l-alert>`](https://github.com/BrightspaceUI/core/tree/main/components/alert), Kaiju has noticed an [accessibility defect (KAIJU-35)](https://desire2learn.atlassian.net/browse/KAIJU-35) where the alert's contents cannot be tabbed through to and read by screen readers.

Kaiju has found a [workaround](https://github.com/Brightspace/smart-curriculum/pull/2126) by adding aria and role attributes to a span containing the alert's child content, but the issue can still be seen when using the alert component as described in the [daylight sandbox](https://daylight.d2l.dev/sandboxes/d2l-alert/).
